Sources_v3/StreamChatUI/ChatChannel/ChatMessage/ChatMessageQuoteBubbleView.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatMessageQuoteBubbleView<ExtraData: ExtraDataTypes>: MessageComposerQuoteBubbleView<ExtraData> {
+    public var isParentMessageSentByCurrentUser: Bool? {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
+    
+    override open func setUpLayout() {
+        super.setUpLayout()
+        
+        // In ChatMessage view is added to superview in `setUpLayout` but all constraint are activated in
+        // `updateContent`
+        // This workaround is preventing ambiguous layout when constraints for this view will be activated
+        // before superview constraints.
+        container.removeFromSuperview()
```

Comment:
Either I'm missing something or it's alarming.
Basically our views from composer and from message are not really compatible.

Reply:
This is strange. `updateContent` is called immediately after `setUpLayout` for this exact reason. All lifecycle methods are called immediately one after each other.